### PR TITLE
Use open-color directly

### PR DESCRIPTION
**Pull request by j-f1**
I also used this opportunity to switch any remaining custom colors to a similar open-color.

### CODE REVIEW

**Comment by j-f1** [comment]
Any idea what’s causing the test fail?

File: src/components/ColorPicker.tsx [hunk 1]
```diff
@@ -124,13 +124,14 @@ const Picker = function ({
         {colors.map((_color, i) => (
           <button
             className="color-picker-swatch"
-            onClick={() => {
+            onClick={(e) => {
+              (e.currentTarget as HTMLButtonElement).focus();
```
Comment (lipis) [inline src/components/ColorPicker.tsx:128]:
```suggestion
            onClick={(event) => {
              (event.currentTarget as HTMLButtonElement).focus();
```

File: src/components/ColorPicker.tsx [hunk 2]
```diff
@@ -250,7 +251,11 @@ export function ColorPicker({
       </div>
       <React.Suspense fallback="">
         {isActive ? (
-          <Popover onCloseRequest={() => setActive(false)}>
+          <Popover
+            onCloseRequest={(e) =>
+              e.target !== pickerButton.current && setActive(false)
```
Comment (lipis) [inline src/components/ColorPicker.tsx:256]:
```suggestion
            onCloseRequest={(event) =>
              event.target !== pickerButton.current && setActive(false)
```

File: src/components/Popover.tsx [hunk 3]
```diff
@@ -38,9 +38,9 @@ export function Popover({
 
   useEffect(() => {
     if (onCloseRequest) {
-      const handler = (e: Event) => {
+      const handler = (e: PointerEvent) => {
         if (!popoverRef.current?.contains(e.target as Node)) {
```
Comment (lipis) [inline src/components/Popover.tsx:42]:
```suggestion
      const handler = (event: PointerEvent) => {
        if (!popoverRef.current?.contains(event.target as Node)) {
```

**Comment by lipis** [comment]
Did you update the snapshots?

**Comment by j-f1** [comment]
Done, although it’s worrying to me that this seemingly superficial change has changed the snapshots.

**Pull request merged**